build.gradle

```diff
@@ -191,6 +191,22 @@ task downloadIntermediary(type: Download) {
 	dest new File(cacheFilesMinecraft, "${minecraft_version}-intermediary.tiny")
 }
 
+task downloadIntermediaryV2(type: Download) {
+	group = buildMappingGroup
+	def url = "https://maven.fabricmc.net/net/fabricmc/intermediary/${minecraft_version}/intermediary-${minecraft_version}-v2.jar"
+	src com.google.common.net.UrlEscapers.urlFragmentEscaper().escape(url)
```

Comment:
This is not right, but it probably doesn't really break here so I don't mind too much. The URL fragment is the `#asd` part. Escaping should always happen while translating from one "language" (arbitrary String blobs) to another (URL path segment), so on the individual strings.

Reply:
This has worked fine for downloading the v1 mappings, as far as I know it’s only needed when the version contains a space or some other char. (April fools version)